### PR TITLE
README

### PR DESCRIPTION
**Pull request by robertkarl**

### CODE REVIEW

**Comment by zackgomez** [comment]
haha ./README

I think the best way to do this is to update the package.json to make 'npm run start' do the right thing

File: README [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# ./README
+npm install
+npm install -g babel
+babel-node server.js
+open http://localhost:3500/
```
Comment (zackgomez) [inline README]: this won't be executed until after the server terminates

Comment (robertkarl) [inline README]: Ooops

**Comment by robertkarl** [comment]
I'll update to do this the node-y way

**Pull request closed without merging**